src/analysis/normed/field/basic.lean

```diff
@@ -453,6 +453,15 @@ nnreal.eq $ by simp
 @[simp] lemma nnnorm_zpow : ∀ (a : α) (n : ℤ), ‖a ^ n‖₊ = ‖a‖₊ ^ n :=
 map_zpow₀ (nnnorm_hom : α →*₀ ℝ≥0)
 
+protected lemma normed_division_ring.dist_inv_inv {z w : α} (hz : z ≠ 0) (hw : w ≠ 0) :
```

Comment:
I think either `dist_inv_inv` or `dist_inv_inv\0` would be a better name

Reply:
Agreed!